spring-xsuaa/src/test/java/com/sap/cloud/security/xsuaa/XsuaaServicePropertySourceFactoryMultipleBindingsTest.java

```diff
@@ -0,0 +1,48 @@
+package com.sap.cloud.security.xsuaa;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.Test;
```

Comment:
same as above junit4 instead of junit5

Reply:
Here I just copied the test cases from XsuaaServicesParserTest, which were written in Junit 4

Reply:
oki 👍🏻